### ISSUE TITLE
FR: Suggest tables from WITH query

### ISSUE THREAD

**Issue by KES777** [dbeaver/dbeaver#4748]
Feature request:

I have query
```sql
WITH consumed_resource AS (
	SELECT 
	  cr.resource_uuid,
	  count(*) OVER ( PARTITION BY cr.resource_uuid, cr.resource_type_id ), -- Count of config changes
	  cr.resource_type_id,
	  cr.amount, 
	  ARRAY_AGG( EXTRACT( DAY FROM cr.startdate)::int ORDER BY cr.startdate ) AS days,
	  max( EXTRACT( DAY FROM cr.startdate)::int ) AS last_used
	FROM consumed_resource cr
	WHERE cr.startdate >= '2018-11-01' AND cr.startdate < '2018-12-01'
	GROUP BY cr.resource_uuid, cr.resource_type_id, cr.amount
), od_ag_group AS (
SELECT od.order_id, ar.parent_id
FROM ( SELECT * FROM order_detail od WHERE order_id = 5831 ) od
FULL JOIN (
	SELECT * FROM allocated_resource ar 
		FULL JOIN consumed_resource cr 
		  ON cr.resource_type_id = ar.resource_type_id AND cr.resource_uuid = ar.resource_uuid
	WHERE ar.parent_id = 11010
) ar ON ar.id = od.allocated_resource_id
GROUP BY od.order_id, ar.parent_id
)
SELECT * FROM od
                ^^^ suggest here `od_ag_group`
```

![__](https://user-images.githubusercontent.com/799021/49638313-7c803d80-fa10-11e8-8817-f9896283d3e3.jpg)

May you suggest `od_ag_group` table alias from WITH query?

**Comment by KES777** [comment]
Also suggest fields from WITH query (if possible):
![__](https://user-images.githubusercontent.com/799021/49638615-5c04b300-fa11-11e8-8c80-8f2360d1c096.jpg)

**Comment by uslss** [comment]
is it reproducible for you against recent version?

**Comment by uslss** [comment]
will be implemented in scope of #13108
